### DISCUSSION TITLE
Personal Results won't work

### DISCUSSION THREAD

**Discussion by AnshumanPradhan009** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#408] [Q&A]
When I am telling to set a reminder it is showing that "Sure. To help you create reminders, I need your permission to allow personal results on this device. You can change this in Assistant settings." after allowing also it is showing just the same

**Comment by Josh65-2201** [comment]
They don't work because of the limitations in the SDK version they use.

**Comment by im-coder-lg** [comment]
@AnshumanPradhan009 I found a solution. Go [<kbd>here</kbd>](https://gist.github.com/im-coder-lg/23fc93cbf983b36b21641563ed4f3d4e), follow the steps and you can see at the screenshots I took too. Real proof it works.
